source/css/sass/mixins/_typography.scss

```diff
@@ -91,7 +91,7 @@
   }
 }
 
-@mixin _label-typography($color, $uppercase: true) {
+@mixin label-typography($color, $uppercase: true) {
```

Comment:
We shouldn't remove the `_` prefix from this name. It's a general programming convention that this prefix tells developers that the thing that's `_`-prefixed is private, i.e. is not part of the public API. In scss terms it means that the mixin must not be used outside of the file it's defined in, i.e. that scss external to `_typography.scss`  shouldn't call it. By removing the prefix, we are committing to supporting this mixin in the public API, and a change to a public API is not something to do lightly.

In the current situation, both the mixins `label-content-typography` and `label-tag-typography` use `_label-typography`, so rather than using `_label-typography` directly from `article-type.scss`, use one of these other two mixins that use it.